WalletWasabi.Daemon/Config.cs

```diff
@@ -127,6 +127,9 @@ public Config(PersistentConfig persistentConfig, string[] cliArgs)
 			[ nameof(CoordinatorIdentifier)] = (
 				"-",
 				GetStringValue("CoordinatorIdentifier", PersistentConfig.CoordinatorIdentifier, cliArgs)),
+			[ nameof(MaxCoordinationFeeRate)] = (
```

Comment:
cack,

imo, I'd also remove _rate_ from `MaxCoordinationFeeRate`.
fee rate always gives the impression it's about mining fees.

and after short discussion it was agreed to change this wording for the GUI option 613df4bbf6ab7d9ed75ded2b51eb4803ac2d3879
so should be here too for consistency, no?

Reply:
I don't think so. I agree it makes sense to remove `Rate` from UI because it's user facing and can lead to confusion, but now we are talking about the internal variable, and technically the name `MaxCoordinationFee` is incorrect, as it's a rate. So I don't want to rename the variable, as no other name would be more correct than this one, and I don't want to rename the CLI parameter either because there is no precedent where a CLI parameter has a different name than the internal variable. I understand the users always fee rate to talk about mempool, but well in this case this is also a fee rate.